contracts/src/Teleporter/TeleporterMessenger.sol

```diff
@@ -36,54 +36,56 @@ contract TeleporterMessenger is ITeleporterMessenger, ReentrancyGuards {
     using SafeERC20 for IERC20;
     using ReceiptQueue for ReceiptQueue.TeleporterMessageReceiptQueue;
 
-    // SentMessageInfo includes the fee information for a given message submitted
-    // to be sent, along with the hash of the message itself.
+    /// @notice SentMessageInfo includes the fee information for a given message submitted
```

Comment:
Could you update these to be:
```
/**
 * @notice ...
 */
```
to match the other multi-line natspec comments here?